apis/notebooks/v1alpha1/generate.sh

```diff
@@ -13,14 +13,23 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
 set -o errexit
 set -o nounset
 set -o pipefail
 
 REPO_ROOT="$(git rev-parse --show-toplevel)"
 cd ${REPO_ROOT}/dev/tools/controllerbuilder
 
-go run . generate-types     --service google.cloud.notebooks.v1     --api-version notebooks.cnrm.cloud.google.com/v1alpha1     --resource NotebooksEnvironment:Environment
+go run . generate-types \
```

Comment:
Can we manually combine the commands instead of removing the previous command?

```
go run . generate-types \
    --service google.cloud.notebooks.v1 \
    --api-version notebooks.cnrm.cloud.google.com/v1alpha1 \
    --resource NotebooksEnvironment:Environment \
    --resource NotebooksInstance:Instance
```

Reply:
We can. From an automated point what do you thing about generate_environment.sh and generate_notebook.sh ?

Reply:
Having a separate `generate-types` command per `--resource` causes conflicts and overwrites. Fundamentally, the issue is not just about the script's name but rather the need to consolidate the `generate-types` command for a specific `--api-version`.

Reply:
Fixed with new code generation